src/skmultiflow/trees/nodes/inactive_learning_node_adaptive_multi_target.py

```diff
@@ -50,9 +50,9 @@ def update_weights(self, X, y, learning_ratio, rht):
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```

Comment:
The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.